backend/src/RedisDriver.js

```diff
@@ -0,0 +1,29 @@
+const Redis = require( 'ioredis' );
+const redisAdapter = require( 'socket.io-redis' );
+const dotenv = require( 'dotenv' );
+dotenv.config( { path: '.env' } );
+
+class RedisDriver {
+	constructor( io ) {
+		this.io = io;
+		this._config = {
+			host: process.env.REDIS_HOST,
+			port: process.env.REDIS_PORT
+		};
+		this._client = this._connect( io );
```

Comment:
This is not good practice to execute the connect method (any method) in the constructor.

Please create an object and then execute connect

```
const redisDirver = new RedisDriver(io);

redisDriver.connect();
```

Reply:
Good to know, I've fixed it.